### PR TITLE
feat: UI Page for Inventory

### PR DESCRIPTION
**Pull request by tiffzeng**
#

## Changes this PR introduces
- Inventory Page for Parts, Materials and Bike Products
- Contains tables (+ table pagination) and tabs to switch between the three sections

## How to test this PR
- run `docker-compose up --build` on the client

## ClickUp reference
Task linked: [CU-mz4u33: UI Page to create and list Bikes along with inventory](https://app.clickup.com/t/mz4u33)

## Notes for reviewers
- Login to the ERP
- Click on the Inventory tab from the menu bar
- Switch from tab to tab / play with the table pagination

## Checklist
- [x] Have I renamed the title of this pull request to follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)?
- [x] I will squash and merge when my PR is approved.
- [x] I have added tests to cover my changes (if applicable).

### CODE REVIEW

**Comment by Mackbellemore** [comment]
Task linked: [CU-kx3wq8 UI Page to create and list raw materials along with inventory](https://app.clickup.com/t/kx3wq8)

**Comment by Mackbellemore** [comment]
Task linked: [CU-mz4u33 UI Page to create and list Bikes along with inventory](https://app.clickup.com/t/mz4u33)

**Comment by gordonpn** [comment]
Could you `git merge origin/develop` because the Netlify deployment is bugged right now. As well as solve the merge conflicts.

File: client/src/components/views/inventory items/Parts.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      <>
+        <Heading size="xl" textAlign="center" mt={5}>
+          No Parts
+        </Heading>
+        <Image
+          src="/images/noResults.png"
+          alt="No results illustration"
+          width="100%"
+          height="100%"
+        />
```
Comment (gordonpn) [inline client/src/components/views/inventory items/Parts.jsx:38]: Now that we are using this Image component in 4 different places, we could extract it into a common place.

And I feel that 100% might be too big?

Comment (tiffzeng) [reply client/src/components/views/inventory items/Parts.jsx:38]: you're right, will look into refactoring

**Review by sujvn — APPROVED** [review]

**Review by sujvn — APPROVED** [review]

**Review by SiuYe — APPROVED** [review]
Good work @KarinKazarian @tiffzeng

File: client/src/components/views/inventory items/Parts.jsx (hunk 1 above)
Comment (Mackbellemore) [reply client/src/components/views/inventory items/Parts.jsx:38]: next sprint gordo

**Review by Mackbellemore — APPROVED** [review]

**Review by Mackbellemore — APPROVED** [review]

**Pull request merged**